pkgs/development/libraries/science/math/cudnn/generic.nix

```diff
@@ -49,7 +44,7 @@ stdenv.mkDerivation rec {
   meta = with stdenv.lib; {
     description = "NVIDIA CUDA Deep Neural Network library (cuDNN)";
     homepage = "https://developer.nvidia.com/cudnn";
-    license = licenses.unfree;
+    license = [ licenses.unfree licenses.bsd3 ];
```

Comment:
Does BSD3 apply to anything brought by this derivation, or is it just the license of Dockerfiles where you have found source URLs?

Reply:
That's just the license in the Dockerfiles repo, which I now realize is _definitely_ not the cudnn license. [It looks like this is the actual license on the NVIDIA site.](http://docs.nvidia.com/deeplearning/sdk/cudnn-sla/index.html)
I can add the license to licenses.nix, something like
```
nvidia-cudnn = {
  fullName  = "NVIDIA SOFTWARE LICENSE AGREEMENT for cuDNN";
  url       = "http://docs.nvidia.com/deeplearning/sdk/pdf/cuDNN-SLA.pdf";
  free      = false;
};
```

[`cudnn` and friends have been posted to AUR with license proprietary](https://aur.archlinux.org/packages/cudnn6/)

Reply:
IMHO just keeping the old `unfree` is fine. Then it looks ready to merge.